### PR TITLE
Remove test specification files

### PR DESCRIPTION
**Pull request by bkorycki**
This PR removes test TOML files because their main use case is currently for specifying content to be used by coffee. The files will be moved to coffee instead.

### CODE REVIEW

File: plugins/standard_tests/newhelm/tests/specifications/bbq.toml
Comment (brianwgoldman) [inline plugins/standard_tests/newhelm/tests/specifications/bbq.toml:1]: Would it be worth copying these over to coffee before deleting them here?

Comment (brianwgoldman) [inline plugins/standard_tests/newhelm/tests/specifications/bbq.toml:1]: Some other stuff we should probably delete / copy to coffee:

* newhelm/test_specifications.py
* tests/test_test_specification.py
* newhelm/tests/specifications/demo_01.toml
* The README files in all `tests/specifications/` folders.

**Review by brianwgoldman — APPROVED** [review]

File: plugins/standard_tests/newhelm/tests/specifications/bbq.toml
Comment (bkorycki) [reply plugins/standard_tests/newhelm/tests/specifications/bbq.toml:1]: Yeah maybe, I'll hold off on merging.

Comment (brianwgoldman) [reply plugins/standard_tests/newhelm/tests/specifications/bbq.toml:1]: > I assume the version is staying with the test object?

My understanding is the only reason we added version to these files was for display purposes, so it should move. It may come back when/if we do programmatic versioning.

**Comment by wpietri** [comment]
Happy to defer to Brian and Daniel on this.

**Pull request merged**
